server/routes/journeys/dailySchedule/handlers/dailyScheduleHandler.ts

```diff
@@ -82,6 +82,7 @@ export default class DailyScheduleHandler implements PageHandler {
           schedule,
           date,
           isPastDay: startOfDay(date) < startOfDay(new Date()),
+          isFutureDay: startOfDay(date) > startOfDay(new Date()),
```

Comment:
Could possibly have unit test coverage here for this ???

Reply:
We could and I will sort that. There is unit test coverage on the nunjucks utils function which checks before/after times, but not on this part as you (correctly) point out.

Reply:
Tests now added.